src/node_modules.cc

```diff
@@ -530,12 +601,31 @@ void BindingData::CreatePerContextProperties(Local<Object> target,
   compile_cache_status_values.push_back(                                       \
       FIXED_ONE_BYTE_STRING(isolate, #status));
   COMPILE_CACHE_STATUS(V)
+#undef V
 
   USE(target->Set(context,
                   FIXED_ONE_BYTE_STRING(isolate, "compileCacheStatus"),
                   Array::New(isolate,
                              compile_cache_status_values.data(),
                              compile_cache_status_values.size())));
+
+  LocalVector<v8::Name> cached_code_type_keys(isolate);
+  LocalVector<Value> cached_code_type_values(isolate);
```

Comment:
For consistency, can you add the `v8::Name`, `v8::External`, and `v8::Null`, etc to the `using v8:....` bits at the top.